### PR TITLE
feat(secnav): implement secondary navigation component

### PR DESCRIPTION
**Pull request by lalithkarikelli**
## Description

Implement Sec Nav (Secondary Navigation).

### What are the relevant story cards/tickets? Any additional PRs or other references?

Jira: SURF1766

## Before you request a review for this PR:

- [x] For UI changes, did you manually test in recent versions of modern browsers (Chrome, Firefox, and Safari)?
- [x] For UI changes, did you manually test in IE11 and legacy Edge?
- [ ] Did you add component tests for any new code?
- [ ] Did you run the component unit tests via `yarn test` to ensure all tests passed?
- [ ] Did you include a screenshot of the component tests?
- [ ] If you changed/added functionality, did you update the demo page and documentation?
- [ ] If needed, did you add or modify the demo test page to test the changed/added functionality?
- [x] Did you assign reviewers?
- [x] In Jira, have you linked to this PR on the ticket(s)?

### CODE REVIEW

File: docs/components/secnav/index.html [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+            <a href="#">
+              <hx-icon type="support"></hx-icon>
+              <p>Support</p>
+            </a>
+  
+            <a href="#" class="disabled" id="billing">
+              <hx-icon type="billing"></hx-icon>
+              <p>Billing</p>
+              <hx-tooltip for="billing" position="bottom-center">
+                You do not have access to this area. Contact an account admin in your organization to request access to
```
Comment (100stacks) [inline docs/components/secnav/index.html:62]: Please break this line at <=80ch.

File: docs/components/secnav/index.html [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+  <section>
+    <header>
+      <h2 id="icon-badge">Nav Menu</h2>
+    </header>
+  
+    <div id="vue-topNavDemo">
+      <div id="hxTopNav">
+        <a class="img-left brand-logo" href="#"> <img src="images/helix-logo.svg" alt="Logo" /></a>
+        <div class="productMenu">
+          <hx-disclosure id="topnav-productMenu" aria-controls="topnavProductDropdown" role="button" aria-expanded="false"
```
Comment (100stacks) [inline docs/components/secnav/index.html:28]: Please break attributes across multiple lines.

File: docs/components/secnav/index.html [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+  components/secnav/test.html: Full Page Demo - Secondary Navigation
+  elements/hx-navigation-control: <hx-navigation-control>
+---
+  {% extends 'component.njk' %}
+
+  {% block page_header %}
+  <p>
+    The {{page.title}} is the secondary navigation menu bar across all breakpoints within the application frame of a HelixUI interface.
+    Use the {{page.title}} to provide access to globally available features.
+  </p>
```
Comment (100stacks) [inline docs/components/secnav/index.html:14]: This appears to be the Top Nav description.

Please change:

**{{page.title}} is a variation of navigation that is used in lieu of left navigation. It establishes a user's location within an application and shows the path forward to accomplish a goal.**

File: docs/components/secnav/index.html [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+            </a>
+  
+            <a href="#">
+              <hx-icon type="account"></hx-icon>
+              <p>Account</p>
+            </a>
+          </a>
+          <div class="straightLine"></div>
+          <div>
+            <hx-disclosure id="topnav-menu" aria-controls="topnavMenuId" role="button" aria-expanded="false"
```
Comment (100stacks) [inline docs/components/secnav/index.html:74]: Please break attributes across multiple lines.

File: docs/components/secnav/index.html [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+              </div>
+            </hx-menu>
+          </div>
+        </div>
+      </div>
+      <!-- Secondary Navigation -->
+      <div id="hxSecNav">
+        <a href="#" class="navLink"> Nav Item One</a>
+        <div class="productMenu">
+          <hx-disclosure id="topnav-productMenu" aria-controls="secNavProductDropdown" role="button" aria-expanded="false"
```
Comment (100stacks) [inline docs/components/secnav/index.html:100]: Please break attributes across multiple lines.

File: docs/components/secnav/index.html [hunk 6]
```diff
@@ -0,0 +1,138 @@
...
+          </hx-disclosure>
+          <hx-menu id="secNavProductDropdown">
+            <div class="sectionHeader">
+              <hx-menuitem role="menuitem" class="menuKey">Cloud Server Backups:</hx-menuitem>
+              <hx-menuitem role="menuitem">Systems</hx-menuitem>
+              <hx-menuitem role="menuitem">Backups</hx-menuitem>
+              <hx-menuitem role="menuitem">Activity</hx-menuitem>
+            </div>
+          </hx-menu>
+          <hx-disclosure id="topnav-menu" aria-controls="secNavProductDropdownTwo" role="button" aria-expanded="false"
```
Comment (100stacks) [inline docs/components/secnav/index.html:113]: Please break attributes across multiple lines.

File: docs/components/secnav/index.html [hunk 7]
```diff
@@ -0,0 +1,138 @@
...
+              <div class="sectionFooter">
+                <button class="hxBtn">Log Out </button>
+              </div>
+            </hx-menu>
+          </div>
+        </div>
+      </div>
+      <!-- Secondary Navigation -->
+      <div id="hxSecNav">
+        <a href="#" class="navLink"> Nav Item One</a>
```
Comment (100stacks) [inline docs/components/secnav/index.html:98]: The text should be all uppercase:

Screen Shot 2020-10-14 at 2 43 42 PM

File: docs/components/secnav/index.html [hunk 8]
```diff
@@ -0,0 +1,138 @@
...
+        <div class="productMenu">
+          <hx-disclosure id="topnav-productMenu" aria-controls="secNavProductDropdown" role="button" aria-expanded="false"
+            class="disabled">
+            <span>Nav Item Two</span>
```
Comment (100stacks) [inline docs/components/secnav/index.html:102]: The text should be all uppercase:

Screen Shot 2020-10-14 at 2 43 42 PM

File: docs/components/secnav/index.html [hunk 9]
```diff
@@ -0,0 +1,138 @@
...
+            <div class="sectionHeader">
+              <hx-menuitem role="menuitem" class="menuKey">Cloud Server Backups:</hx-menuitem>
+              <hx-menuitem role="menuitem">Systems</hx-menuitem>
+              <hx-menuitem role="menuitem">Backups</hx-menuitem>
+              <hx-menuitem role="menuitem">Activity</hx-menuitem>
+            </div>
+          </hx-menu>
+          <hx-disclosure id="topnav-menu" aria-controls="secNavProductDropdownTwo" role="button" aria-expanded="false"
+            class="disabled">
+            <span>Nav Item Three</span>
```
Comment (100stacks) [inline docs/components/secnav/index.html:115]: The text should be all uppercase:

Screen Shot 2020-10-14 at 2 43 42 PM

File: docs/components/secnav/index.html [hunk 10]
```diff
@@ -0,0 +1,138 @@
...
+            </div>
+          </hx-menu>
+        </div>
+      </div>
+      <footer>
+        <pre><code v-text="snippet"></code></pre>
+      </footer>
+    </div>
+  </section>
+  {% endblock %}
```
Comment (100stacks) [inline docs/components/secnav/index.html:138]: Please add a blank line.

**Review by 100stacks — CHANGES_REQUESTED** [review]

File: docs/components/secnav/index.html [hunk 11]
```diff
@@ -0,0 +1,138 @@
+---
+title: Sec Nav
+minver: 1.1.0
+also:
+  components/secnav/test.html: Full Page Demo - Secondary Navigation
+  elements/hx-navigation-control: <hx-navigation-control>
```
[the remark this answers is not in the document]
Comment (lalithkarikelli) [reply docs/components/secnav/index.html:6]: Changes have been done, please review.

File: docs/components/secnav/test.html [hunk 12]
```diff
@@ -0,0 +1,117 @@
...
+title: Testing - Secondary Navigation
+---
+{% extends 'no-frame.njk' %}
+
+{% block content %}
+<div id="vue-topNavDemo">
+    <div id="hxTopNav">
+        <a class="img-left brand-logo" href="#"> <img src="images/helix-logo.svg" alt="Logo" /></a>
+        <div class="productMenu">
+          <hx-disclosure id="topnav-productMenu" aria-controls="topnavProductDropdown" role="button" aria-expanded="false"
```
[the remark this answers is not in the document]
Comment (lalithkarikelli) [reply docs/components/secnav/test.html:11]: Changes have been done, please review.

File: docs/components/secnav/test.html [hunk 13]
```diff
@@ -0,0 +1,117 @@
...
+            <a href="#">
+              <hx-icon type="support"></hx-icon>
+              <p>Support</p>
+            </a>
+  
+            <a href="#" class="disabled" id="billing">
+              <hx-icon type="billing"></hx-icon>
+              <p>Billing</p>
+              <hx-tooltip for="billing" position="bottom-center">
+                You do not have access to this area. Contact an account admin in your organization to request access to
```
[the remark this answers is not in the document]
Comment (lalithkarikelli) [reply docs/components/secnav/test.html:45]: Changes have been done, please review.

File: docs/components/secnav/test.html [hunk 14]
```diff
@@ -0,0 +1,117 @@
...
+            </a>
+  
+            <a href="#">
+              <hx-icon type="account"></hx-icon>
+              <p>Account</p>
+            </a>
+          </a>
+          <div class="straightLine"></div>
+          <div>
+            <hx-disclosure id="topnav-menu" aria-controls="topnavMenuId" role="button" aria-expanded="false"
```
[the remark this answers is not in the document]
Comment (lalithkarikelli) [reply docs/components/secnav/test.html:57]: Changes have been done, please review.

File: docs/components/secnav/test.html [hunk 15]
```diff
@@ -0,0 +1,117 @@
...
+              </div>
+            </hx-menu>
+          </div>
+        </div>
+      </div>
+      <!-- Secondary Navigation -->
+      <div id="hxSecNav">
+          <a href="#" class="navLink"> Nav Item One</a>
+          <div class="productMenu">
+            <hx-disclosure id="topnav-productMenu" aria-controls="secNavProductDropdown" role="button" aria-expanded="false"
```
[the remark this answers is not in the document]
Comment (lalithkarikelli) [reply docs/components/secnav/test.html:83]: Changes have been done, please review.

File: docs/components/secnav/test.html [hunk 16]
```diff
@@ -0,0 +1,117 @@
...
+            </hx-disclosure>
+            <hx-menu id="secNavProductDropdown">
+              <div class="sectionHeader">
+                <hx-menuitem role="menuitem" class="menuKey">Cloud Server Backups:</hx-menuitem>
+                <hx-menuitem role="menuitem">Systems</hx-menuitem>
+                <hx-menuitem role="menuitem">Backups</hx-menuitem>
+                <hx-menuitem role="menuitem">Activity</hx-menuitem>
+              </div>
+            </hx-menu>
+            <hx-disclosure id="topnav-menu" aria-controls="secNavProductDropdownTwo" role="button" aria-expanded="false"
```
[the remark this answers is not in the document]
Comment (lalithkarikelli) [reply docs/components/secnav/test.html:96]: Changes have been done, please review.

File: docs/components/secnav/test.html [hunk 17]
```diff
@@ -0,0 +1,117 @@
...
+              <div class="sectionBody">
+                <hx-menuitem role="menuitem" class="menuKey">Object Storage and CDN :</hx-menuitem>
+                <hx-menuitem role="menuitem" class="menuValue">Files</hx-menuitem>
+                <hx-menuitem role="menuitem" class="menuValue">CDN</hx-menuitem>
+              </div>
+            </hx-menu>
+          </div>
+        </div>
+</div>
+{% endblock %}
```
[the remark this answers is not in the document]
Comment (lalithkarikelli) [reply docs/components/secnav/test.html:117]: Changes have been done, please review.

File: docs/components/secnav/index.html (hunk 1 above)
Comment (lalithkarikelli) [reply docs/components/secnav/index.html:62]: Changes have been done, please review.

File: docs/components/secnav/index.html (hunk 2 above)
Comment (lalithkarikelli) [reply docs/components/secnav/index.html:28]: Changes have been done, please review.

File: docs/components/secnav/index.html (hunk 3 above)
Comment (lalithkarikelli) [reply docs/components/secnav/index.html:14]: Changes have been done, please review.

File: docs/components/secnav/index.html (hunk 4 above)
Comment (lalithkarikelli) [reply docs/components/secnav/index.html:74]: Changes have been done, please review.

File: docs/components/secnav/index.html (hunk 5 above)
Comment (lalithkarikelli) [reply docs/components/secnav/index.html:100]: Changes have been done, please review.

File: docs/components/secnav/index.html (hunk 6 above)
Comment (lalithkarikelli) [reply docs/components/secnav/index.html:113]: Changes have been done, please review.

File: docs/components/secnav/index.html (hunk 7 above)
Comment (lalithkarikelli) [reply docs/components/secnav/index.html:98]: Changes have been done, please review.

File: docs/components/secnav/index.html (hunk 8 above)
Comment (lalithkarikelli) [reply docs/components/secnav/index.html:102]: Changes have been done, please review.

File: docs/components/secnav/index.html (hunk 9 above)
Comment (lalithkarikelli) [reply docs/components/secnav/index.html:115]: Changes have been done, please review.

File: docs/components/secnav/index.html (hunk 10 above)
Comment (lalithkarikelli) [reply docs/components/secnav/index.html:138]: Changes have been done, please review.

File: src/scss/components/secnav/index.scss [hunk 18]
```diff
@@ -0,0 +1,208 @@
...
+
+    a.selected {
+      border-bottom: 0.25rem solid $red-900;
+
+      > hx-icon, p {
+        color: $gray-25;
+      }
+    }
+  }
+}
```
[the remark this answers is not in the document]
Comment (lalithkarikelli) [reply src/scss/components/secnav/index.scss]: Changes have been done, please review.

File: src/scss/components/secnav/index.scss [hunk 19]
```diff
@@ -0,0 +1,208 @@
...
+        background-color: $gray-1000;
+      }
+
+      &:active {
+        color: $cyan-700;
+        border-color: $cyan-700;
+      }
+    }
+  }
+}
```
[the remark this answers is not in the document]
Comment (lalithkarikelli) [reply src/scss/components/secnav/index.scss]: Changes have been done, please review.

**Review by BY00565233 — APPROVED** [review]
Dev LGTM

File: docs/components/secnav/index.html [hunk 20]
```diff
@@ -0,0 +1,138 @@
...
+also:
+  components/secnav/test.html: Full Page Demo - Secondary Navigation
+---
+  {% extends 'component.njk' %}
+
+  {% block page_header %}
+  <p>
+      {{page.title}} is a variation of navigation that is used in lieu of left navigation. It establishes a user's 
+      location within an application and shows the path forward to accomplish a goal.
+  </p>
```
Comment (100stacks) [inline docs/components/secnav/index.html:13]: Please update:

```html
  <p>
    {{page.title}} (i.e., secondary navigation) is a variation of navigation
    that is used in lieu of left navigation. It establishes a user's location
    within an application.  Unless a firm implementation constraint requires
    {{page.title}}, use the left navigation component for web applications.
  </p>
```

**Review by 100stacks — CHANGES_REQUESTED** [review]

File: src/scss/components/secnav/index.scss [hunk 21]
```diff
@@ -108,10 +15,15 @@
     padding: 1rem 1.5rem;
     color: $gray-400;
     font-size: 0.875rem;
+    text-transform: uppercase;
   }
 }
 
-#topnav-menu, #topnav-productMenu {
+#secNavMenu {
```
[the remark this answers is not in the document]
Comment (lalithkarikelli) [reply src/scss/components/secnav/index.scss]: Changes have been done, please review.

File: src/scss/components/secnav/index.scss [hunk 22]
```diff
@@ -138,7 +50,7 @@
   }
 }
 
-#topnavMenuId, #topnavProductDropdown, #secNavProductDropdown, #secNavProductDropdownTwo {
+#secNavProductDropdown, #secNavProductDropdownTwo {
```
[the remark this answers is not in the document]
Comment (lalithkarikelli) [reply src/scss/components/secnav/index.scss]: Changes have been done, please review.

File: src/scss/components/secnav/index.scss [hunk 23]
```diff
@@ -108,10 +15,15 @@
...
+    text-transform: uppercase;
   }
 }
 
-#topnav-menu, #topnav-productMenu {
+#secNavMenu {
+  text-transform: uppercase;
+}
+
+#secNavMenu {
```
[the remark this answers is not in the document]
Comment (lalithkarikelli) [reply src/scss/components/secnav/index.scss]: Changes have been done, please review.

File: docs/components/secnav/test.html [hunk 24]
```diff
@@ -0,0 +1,118 @@
...
+        class="disabled">
+        <span>Nav Item Two</span>
+        <hx-icon class="hxPrimary" type="angle-down"></hx-icon>
+      </hx-disclosure>
+      <hx-menu id="secNavProductDropdown">
+        <div class="sectionHeader">
+          <hx-menuitem role="menuitem" class="menuKey">Cloud Server Backups:</hx-menuitem>
+          <hx-menuitem role="menuitem">Systems</hx-menuitem>
+          <hx-menuitem role="menuitem">Backups</hx-menuitem>
+          <hx-menuitem role="menuitem">Activity</hx-menuitem>
```
[the remark this answers is not in the document]
Comment (lalithkarikelli) [reply docs/components/secnav/test.html]: Changes have been done, please review.

File: docs/components/secnav/test.html [hunk 25]
```diff
@@ -0,0 +1,118 @@
...
+        <div class="sectionHeader">
+          <hx-menuitem role="menuitem" class="menuKey">Block Storage:</hx-menuitem>
+          <hx-menuitem role="menuitem" class="menuValue">Block Storage Volumes</hx-menuitem>
+          <hx-menuitem role="menuitem" class="menuValue">Snapshots</hx-menuitem>
+        </div>
+        <hr class="divider">
+        <div class="sectionBody">
+          <hx-menuitem role="menuitem" class="menuKey">Object Storage and CDN :</hx-menuitem>
+          <hx-menuitem role="menuitem" class="menuValue">Files</hx-menuitem>
+          <hx-menuitem role="menuitem" class="menuValue">CDN</hx-menuitem>
```
[the remark this answers is not in the document]
Comment (lalithkarikelli) [reply docs/components/secnav/test.html]: Changes have been done, please review.

File: docs/components/secnav/secnav-demo.js [hunk 26]
```diff
@@ -0,0 +1,151 @@
...
+            },
+            snippet: function () {
+                return Util.snippet(`
+                <div id="hxTopNav">
+                <a class="img-left brand-logo" href="#"> <img src="images/helix-logo.svg" alt="Logo" /></a>
+                <div class="productMenu">
+                  <hx-disclosure id="topnav-productMenu" 
+                    aria-controls="topnavProductDropdown" 
+                    role="button" aria-expanded="false"
+                    class="disabled">
```
[the remark this answers is not in the document]
Comment (lalithkarikelli) [reply docs/components/secnav/secnav-demo.js]: Changes have been done, please review.

File: docs/components/secnav/index.html (hunk 20 above)
Comment (lalithkarikelli) [reply docs/components/secnav/index.html]: Changes have been done, please review.

File: docs/components/secnav/index.html [hunk 27]
```diff
@@ -0,0 +1,152 @@
...
+    {{page.title}}, use the left navigation component for web applications.
+  </p>
+  {% endblock %}
+
+  {% block content %}
+
+  <section>
+    <header>
+      <h2 id="icon-badge">Secondary Navigation</h2>
+    </header>
```
Comment (100stacks) [inline docs/components/secnav/index.html:23]: Please update:

```html
    <header>
      <h2 id="secondary-navigation">Secondary Navigation</h2>
      <h3>
        See the snippet below in isolation with the
        <a href="components/secnav/test.html">full-page</a> demo.
      </h3>
      <br />
    </header>
```

**Review by 100stacks — CHANGES_REQUESTED** [review]

File: docs/components/secnav/index.html [hunk 28]
```diff
@@ -0,0 +1,152 @@
...
+              <hr class="divider">
+              <div class="sectionFooter">
+                <button class="hxBtn">Log Out </button>
+              </div>
+            </hx-menu>
+          </div>
+        </div>
+      </div>
+      <!-- Secondary Navigation -->
+      <div id="hxSecNav">
```
[the remark this answers is not in the document]
Comment (lalithkarikelli) [reply docs/components/secnav/index.html]: Changes have been done, please review.

File: src/scss/components/secnav/index.scss [hunk 29]
```diff
@@ -0,0 +1,117 @@
+// ----- BASE ELEMENTS ---------------
+@import "vars";
+@import "./config";
+
+#hxSecNav {
+  overflow: hidden;
+  width: 100%;
+  background-color: $gray-975;
+  height: 3.5rem;
```
[the remark this answers is not in the document]
Comment (lalithkarikelli) [reply src/scss/components/secnav/index.scss]: Changes have been done, please review.

File: src/scss/components/secnav/index.scss [hunk 30]
```diff
@@ -0,0 +1,117 @@
...
+
+  .navLink {
+    float: left;
+    width: 10rem;
+    height: 3.5rem;
+    padding: 1rem 1.5rem;
+    color: $gray-400;
+    font-size: 0.875rem;
+    text-transform: uppercase;
```
[the remark this answers is not in the document]
Comment (lalithkarikelli) [reply src/scss/components/secnav/index.scss]: Changes have been done, please review.

File: src/scss/components/secnav/index.scss [hunk 31]
```diff
@@ -0,0 +1,117 @@
...
+
+  .secNavMenu > hx-disclosure {
+    background-color: transparent;
+    line-height: 1;
+    font-size: 0.875rem;
+    padding: 0.625rem 1.25rem 0;
+    color: $gray-400;
+    height: 3.5rem;
+    width: auto;
+    text-transform: uppercase;
```
[the remark this answers is not in the document]
Comment (lalithkarikelli) [reply src/scss/components/secnav/index.scss]: Changes have been done, please review.

File: src/scss/components/secnav/index.scss [hunk 32]
```diff
@@ -0,0 +1,117 @@
...
+
+    > hx-icon {
+      width: 0.75rem;
+      color: $gray-400;
+      margin-top: 0.75rem;
+      margin-left: 0.75rem;
+      font-size: 0.875rem;
```
[the remark this answers is not in the document]
Comment (lalithkarikelli) [reply src/scss/components/secnav/index.scss]: Changes have been done, please review.

File: src/scss/components/secnav/index.scss [hunk 33]
```diff
@@ -0,0 +1,117 @@
...
+    }
+  }
+
+  .secNavMenu > hx-menu {
+    background-color: $gray-1000;
+    min-width: 10.5rem;
+    color: $gray-0;
+    border: 0.063rem solid $gray-1000;
+    top: 14.5rem;
+    padding: 0;
```
[the remark this answers is not in the document]
Comment (lalithkarikelli) [reply src/scss/components/secnav/index.scss]: Changes have been done, please review.

File: src/scss/components/secnav/index.scss [hunk 34]
```diff
@@ -0,0 +1,117 @@
...
+      @include hxSecNavControl(disabled);
+
+      color: $gray-600;
+    }
+
+    .menuKey {
+      font-size: 0.75rem;
+      color: $gray-600;
+      text-transform: none;
+      line-height: 1;
```
[the remark this answers is not in the document]
Comment (lalithkarikelli) [reply src/scss/components/secnav/index.scss]: Changes have been done, please review.

File: src/scss/components/secnav/index.scss [hunk 35]
```diff
@@ -0,0 +1,117 @@
...
+    }
+
+    .menuValue {
+      color: $gray-400;
+      text-transform: none;
+      line-height: 1;
```
[the remark this answers is not in the document]
Comment (lalithkarikelli) [reply src/scss/components/secnav/index.scss]: Changes have been done, please review.

File: src/scss/components/secnav/index.scss [hunk 36]
```diff
@@ -0,0 +1,117 @@
...
+    }
+
+    .divider {
+      height: 0.063rem;
+      border-color: $gray-800;
```
[the remark this answers is not in the document]
Comment (lalithkarikelli) [reply src/scss/components/secnav/index.scss]: Changes have been done, please review.

File: src/scss/components/secnav/index.scss [hunk 37]
```diff
@@ -0,0 +1,117 @@
...
+    .sectionFooter {
+      padding: 1rem 1.25rem 1.25rem;
+
+      .hxBtn {
+        width: -webkit-fill-available;
+
+        &:hover {
+          color: $cyan-100;
+          border-color: $cyan-100;
+          background-color: $gray-1000;
```
[the remark this answers is not in the document]
Comment (lalithkarikelli) [reply src/scss/components/secnav/index.scss]: Changes have been done, please review.

File: src/scss/components/secnav/index.scss [hunk 38]
```diff
@@ -0,0 +1,117 @@
...
+        }
+
+        &:active {
+          color: $cyan-700;
+          border-color: $cyan-700;
```
[the remark this answers is not in the document]
Comment (lalithkarikelli) [reply src/scss/components/secnav/index.scss]: Changes have been done, please review.

File: src/scss/components/secnav/mixins/_secNavControl.scss [hunk 39]
```diff
@@ -0,0 +1,56 @@
...
+    }
+  }
+}
+
+@mixin hxSecNavControl(
+  $state: null,
+  $pseudo-state: null,
+) {
+  @if $state == error {
+    @include __secNavControl--error($pseudo-state);
```
[the remark this answers is not in the document]
Comment (lalithkarikelli) [reply src/scss/components/secnav/mixins/_secNavControl.scss]: Changes have been done, please review.

File: docs/components/secnav/secnav-demo.js [hunk 40]
```diff
@@ -0,0 +1,158 @@
...
+                      <hr class="divider">
+                      <div class="sectionFooter">
+                        <button class="hxBtn">Log Out </button>
+                      </div>
+                    </hx-menu>
+                  </div>
+                </div>
+              </div>
+              <!-- Secondary Navigation -->
+              <div id="hxSecNav">
```
[the remark this answers is not in the document]
Comment (lalithkarikelli) [reply docs/components/secnav/secnav-demo.js]: Changes have been done, please review.

File: docs/components/secnav/test.html [hunk 41]
```diff
@@ -0,0 +1,132 @@
...
+          <hr class="divider">
+          <div class="sectionFooter">
+            <button class="hxBtn">Log Out </button>
+          </div>
+        </hx-menu>
+      </div>
+    </div>
+  </div>
+  <!-- Secondary Navigation -->
+  <div id="hxSecNav">
```
[the remark this answers is not in the document]
Comment (lalithkarikelli) [reply docs/components/secnav/test.html]: Changes have been done, please review.

File: src/scss/components/secnav/index.scss [hunk 42]
```diff
@@ -17,103 +17,100 @@
...
-  &:hover {
-    background-color: $gray-900;
-    color: $gray-0 !important;
-  }
-}
+    padding: 0.625rem 1.25rem 0;
+    color: $gray-400;
+    height: 3.5rem;
+    width: auto;
+    text-transform: uppercase;
```
[the remark this answers is not in the document]
Comment (lalithkarikelli) [reply src/scss/components/secnav/index.scss]: Changes have been done, please review.

File: src/scss/components/secnav/index.scss [hunk 43]
```diff
@@ -17,103 +17,100 @@
     font-size: 0.875rem;
     text-transform: uppercase;
```
[the remark this answers is not in the document]
Comment (lalithkarikelli) [reply src/scss/components/secnav/index.scss]: Changes have been done, please review.

File: src/scss/components/secnav/index.scss [hunk 44]
```diff
@@ -17,103 +17,100 @@
...
-  }
+    .menuValue {
+      color: $gray-400;
+      text-transform: none;
+      line-height: 1;
+    }
 
-  .sectionFooter {
-    padding: 1rem 1.25rem 1.25rem;
+    .divider {
```
[the remark this answers is not in the document]
Comment (lalithkarikelli) [reply src/scss/components/secnav/index.scss]: Changes have been done, please review.

File: src/scss/components/secnav/index.scss [hunk 45]
```diff
@@ -17,103 +17,100 @@
...
-  color: $gray-400;
-  height: 3.5rem;
-  width: auto;
-
-  > hx-icon {
-    width: 0.75rem;
-    color: $gray-400;
-    margin-top: 0.75rem;
-    margin-left: 0.75rem;
+  .secNavMenu > hx-disclosure {
```
[the remark this answers is not in the document]
Comment (lalithkarikelli) [reply src/scss/components/secnav/index.scss]: Changes have been done, please review.

File: src/scss/components/secnav/index.scss [hunk 46]
```diff
@@ -17,103 +17,100 @@
...
 
     &:hover {
       background-color: $gray-900;
       color: $gray-0 !important;
     }
   }
 
-  hx-menuitem.disabled {
-    @include hxSecNavControl(disabled);
+  .secNavMenu > hx-menu {
```
[the remark this answers is not in the document]
Comment (lalithkarikelli) [reply src/scss/components/secnav/index.scss]: Changes have been done, please review.

File: docs/components/secnav/index.html (hunk 27 above)
Comment (lalithkarikelli) [reply docs/components/secnav/index.html]: Changes have been done, please review.

**Review by 100stacks — APPROVED** [review]
Great job! 

Dev LGTM

**Pull request merged**